src/utils/commonUtils.ts

```diff
@@ -185,6 +185,29 @@ export const getTvlImpact = (tvl: number): number => {
     return 5;
 };
 
+export const getLongevityScore = (days: number): number => {
```

Comment:
should we make a risk util.ts? founds like we will keep adding more and more to commons that may be fitting for a /utils/score or /utils/risk file

Reply:
Done. I created utils/risk file